src/drivers/qtouch/qtouch.c

```diff
@@ -273,6 +273,29 @@ static touch_ret_t touch_sensors_config(void)
 	return (touch_ret);
 }
 
+
+/*
+ * Force calibrate
+ *
+ * Call this function when the user "probably" isn't touching the device to reset all the
+ * calibration values.
+ * TODO: Disable if key is active
+ */
+//#include <screen.h>
+void qtouch_force_calibrate(void) {
+    qtm_touch_key_data_t* key;
+    for (uint16_t i = 0u; i < DEF_NUM_CHANNELS; i++) {
+        key = &qtlib_key_data_set1[i];
+        uint16_t value = key->node_data_struct_ptr->node_acq_signals;
+        uint16_t reference = key->channel_reference;
+        int32_t diff = (int32_t)reference - (int32_t)value;
+        //screen_sprintf_debug(100, "%u %u %d", value, reference, diff);
+        if(diff > 10 || diff < -10){
```

Comment:
nit would be to make the `10` a macro 

also remove the debug printing before merge

Reply:
Haven't tested this PR, so this is speculation:

The scenario of touching a sensor on plugin should only give `reference` > `value`. So it is safer to only re-calibrate on that scenario. Then the other scenario would occur if a touch is happening, and so re-calibrating then could lead to a 'dead' sensor later.

Reply:
I agree, changed to only check `reference > value` and also check key state to avoid spurious triggers.